### PR TITLE
Add CreateBucket method

### PR DESCRIPTION
**Pull request by prdlk**
## Changes

* Item 1
* Item 2

## Checklist

* [ ] Unit tests
* [ ] Documentation

## References *(optional)*

Fixes
Connects

 Mention [stepsize] in a comment if you'd like to report some technical debt. See examples here.

### CODE REVIEW

**Comment by prdlk** [comment]
Added create bucket method to bind

File: pkg/motor/interface.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ type MotorNode interface {
 	// Buckets
 
 	// Creates a new bucket with the defined properties in the request. Returns and instance of `bucket`. before returning both content and buckets are resolved.
-	CreateBucket(context.Context, mt.CreateBucketRequest) (bucket.Bucket, error)
+	CreateBucket(mt.CreateBucketRequest) (*mt.CreateBucketResponse, bucket.Bucket, error)
```
Comment (ntindle) [inline pkg/motor/interface.go:54]: Will this one break speedway?

File: pkg/motor/create_bucket.go [hunk 2]
```diff
@@ -13,41 +12,41 @@ import (
 	bt "github.com/sonr-io/sonr/x/bucket/types"
 )
 
-func (mtr *motorNodeImpl) CreateBucket(ctx context.Context, request mt.CreateBucketRequest) (bucket.Bucket, error) {
```
Comment (ntindle) [inline pkg/motor/create_bucket.go:16]: Will this change break speedway

Comment (mcjcloud) [reply pkg/motor/create_bucket.go:16]: It'll just require changes next time speedway upgrades to this version

File: pkg/motor/create_bucket.go [hunk 3]
```diff
@@ -59,5 +58,7 @@ func (mtr *motorNodeImpl) CreateBucket(ctx context.Context, request mt.CreateBuc
 
 	mtr.AddBucketServiceEndpoint(mtr.GetClient().GetRPCAddress(), cbresp.WhereIs.Did)
 
-	return b, nil
+	return &mt.CreateBucketResponse{
+		Did: cbresp.WhereIs.Did,
```
Comment (mcjcloud) [inline pkg/motor/create_bucket.go]: Would it be useful to include the `WhereIs` here as well?

**Review by mcjcloud — APPROVED** [review]

File: pkg/motor/create_bucket.go (hunk 2 above)
Comment (joshLong145) [reply pkg/motor/create_bucket.go:16]: Yea, speedway won't break if it doesn't update to it.

**Review by joshLong145 — APPROVED** [review]

File: pkg/motor/create_bucket.go (hunk 3 above)
Comment (prdlk) [reply pkg/motor/create_bucket.go]: Yeah imma update the response proto to include and merge @mcjcloud

File: pkg/motor/interface.go (hunk 1 above)
Comment (prdlk) [reply pkg/motor/interface.go:54]: The motor impl itself has a context, IDK why we're passing it into methods

**Pull request merged**